app/.env.example

```diff
@@ -1,4 +1,6 @@
 VITE_APP_NAME="Luminary App"
+VITE_BRAND_NAME="Some brand name"
+VITE_LOGO_SVG='<svg></svg>'
```

Comment:
Change to VITE_LOGO and VITE_LOGO_SMALL. Add comments in the example env file to explain the different uses. The variable value should be the relative path or absolute http link to the file.